agent/authenticator/authenticator_suite_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
```suggestion
// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Reply:
Fixed for this and other newly created files. For existing files it is still 2021.